.eslintrc

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Also maybe unnecessary since there's no ESLint in this project, but recommended by Sanity